java/ql/lib/change-notes/2022-09-13-Member-getQualifier.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
```

Comment:
OOI: Isn't this a breaking change?
A lot of queries could stop working, simply because they defined their classes using `Member.getQualifiedName() = foo` in the char. predicate.
As a user, I would consider this a breaking change.

Reply:
Fair point, I'll change it. Thanks!